src/main/java/org/sopt/lequuServer/domain/book/service/BookService.java

```diff
@@ -27,4 +33,51 @@ public BookCreateResponseDto createBook(Book book, Member member) {
 
         return BookCreateResponseDto.of(createdBook);
     }
+
+    // 정해진 기간의 레큐노트를 가져와 많이 부착된 순서로 레큐북 정렬하는 로직
+    public List<Book> sortBooksDesc(LocalDateTime startDate, LocalDateTime endDate) {
+        return bookRepository.findBooksMonth(startDate, endDate)
+            .stream()
+            .sorted(Comparator.comparingInt(book -> -book.getNotes().size())) // -를 붙여 내림차순 정렬
+            .collect(Collectors.toList());
+    }
+
+    // 최근 3달을 확인하는데 최근 순으로 먼저 인기레큐북에 추가할 수 있는 로직
+    // 현재를 0이라 하면 -1 ~ 0(최근 한달), -2 ~ -1(지지난달 부터 지난달), -3 ~ -2(지지지난달 부터 지지난달)
+    public List<Book> getPopularBooks() {
+        List<Book> popularBooks = new ArrayList<>();
+        LocalDateTime today = LocalDateTime.now();
+
+        // 최근 3달 확인
+        for (int i = 0; i < 3; i++) {
+            LocalDateTime endDate = today.minusMonths(i);
+            LocalDateTime startDate = endDate.minusMonths(1);
+            List<Book> books = sortBooksDesc(startDate, endDate);
```

Comment:
```suggestion
    public List<Book> sortBooksDesc(LocalDateTime today, int i) {
        return bookRepository.findBooksMonth(today.minusMonths(i+1), today.minusMonths(i))
            .stream()
            .sorted(Comparator.comparingInt(book -> -book.getNotes().size())) // -를 붙여 내림차순 정렬
            .collect(Collectors.toList());
    }

    // 최근 3달을 확인하는데 최근 순으로 먼저 인기레큐북에 추가할 수 있는 로직
    // 현재를 0이라 하면 -1 ~ 0(최근 한달), -2 ~ -1(지지난달 부터 지난달), -3 ~ -2(지지지난달 부터 지지난달)
    public List<Book> getPopularBooks() {
        List<Book> popularBooks = new ArrayList<>();

        // 최근 3달 확인
        for (int i = 0; i < 3; i++) {
            List<Book> books = sortBooksDesc(LocalDateTime.now(), i);
```
[P3] 이거는 그냥 지저분한 필드 값을 메서드 내로 옮겨버리면 이런 형태도 나올 수 있겠다 싶어서 첨언합니다!
보고 이런 스타일 맘에 드시면 적용하면 되구, 아니면 넘어가도 좋습니다 ㅎ ㅎ